### PR TITLE
Support Rails 7

### PR DESCRIPTION
**Pull request by erikaxel**
Rails 7 only supports Ruby 2.7+, so this commit also adds support for only running tests on a subset of different Ruby versions pr supported Rails version.

### CODE REVIEW

**Comment by erikaxel** [comment]
Hi, this should now be ready for merge. Im sorry if my repeated tries at getting Github Actions to work pinged you unnecessarily.

**Review by tachyons — APPROVED** [review]

**Comment by erikaxel** [comment]
@jenseng If you could merge this I would be happy to see if I can add testing support for more recent Rubies such as 3.0 and 3.1 as well.

**Comment by AlfonsoUceda** [comment]
@erikaxel thanks for your work.

Is there any news on this? It would be nice to support rails 7 and ruby 3.1

File: .github/workflows/workflow.yml
Comment (joeyparis) [inline .github/workflows/workflow.yml:33]: It probably wouldn't hurt to add `3.1`/`3.2` here as well since those are preferred for `Rails 7`. Anything to get Rails 7 support first though.

**Comment by pil0u** [comment]
Hey, migrating to Rails 7 as well, this PR is very welcome :+1:

**Comment by AlfonsoUceda** [comment]
@joeyparis are you the current maintainer of the repo?

File: .github/workflows/workflow.yml
Comment (erikaxel) [reply .github/workflows/workflow.yml:33]: I think it makes sense to add support for 3.0 and 3.1 as a separate PR? I'm happy to do that as soon as this is merged. We have used both 3.0 and 3.1 with hair_trigger so that should be no problem

Comment (joeyparis) [reply .github/workflows/workflow.yml:33]: That makes sense to me, especially if there's a chance adding those to this PR delays getting it merged in.

**Comment by joeyparis** [comment]
@AlfonsoUceda unfortunately not

**Comment by AlfonsoUceda** [comment]
@jenseng could you take a look?

**Comment by jenseng** [comment]
Thanks @erikaxel! Sorry for the slow turnaround, I'll push out a new version in a bit

**Comment by jenseng** [comment]
v0.2.25 should support rails 7

**Comment by AlfonsoUceda** [comment]
Thank you @jenseng

File: .github/workflows/workflow.yml
Comment (AlfonsoUceda) [reply .github/workflows/workflow.yml:33]: @erikaxel could you create a PR for rubies 3.0 and 3.1?

**Pull request merged**
